### PR TITLE
mobile-CI-CD

### PR DESCRIPTION
**Pull request by Mnoble-19**
#### Summary of Changes (What does this PR do?)
Adds CI/CD to android mobile application

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* close a pull request on the staging branch of the repository

#### What are the relevant tickets?
- [MOB-21/MOB-134](https://airqoteam.atlassian.net/browse/MOB-134?atlOrigin=eyJpIjoiNjM3YTBkZDU2NDhmNDc2OTg5N2Y4MDUwYTExNDgxNDciLCJwIjoiaiJ9)

####Screenshots
Screenshot 2022-11-17 at 10 33 32

### CODE REVIEW

File: mobile/android/fastlane/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+metadata/
```
Comment (NoahNsimbe) [inline mobile/android/fastlane/.gitignore:1]: Thanks @Mnoble-19 ... This is a amazing.
For the metadata folder

1. What are its contents?
2. Where can a developer find the content?
3. Has the readme file been updated to incorporate a description and usage of this folder?

Comment (Mnoble-19) [reply mobile/android/fastlane/.gitignore:1]: Hi @NoahNsimbe , the metadata folder contains images, screenshots of the app that are in the PlayStore. It's created as you initialise the Fastlane framework, but it's not something that's needed for the build action to occur.

Comment (NoahNsimbe) [reply mobile/android/fastlane/.gitignore:1]: okay thanks @Mnoble-19

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**